lib/utils/type-tracker/jsdoc/index.ts

```diff
@@ -90,7 +90,7 @@ export class JSDoc {
         }
         const params = (this.params = new JSDocParams())
         for (const param of this.genTags("param")) {
-            const paths: string[] = (param.name || "").split(/\./gu)
+            const paths: string[] = (param.name || "").split(/\./u)
```

Comment:
Why was the `g` flag removed here?

Reply:
`split` seems to search everything without the `g` flag. I knew it for the first time when I looked it up when making rule.

Reply:
Wow. I didn't know that.